### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by sashako**
screen shot 2017-03-20 at 2 47 22 pm
screen shot 2017-03-20 at 2 47 42 pm
screen shot 2017-03-20 at 2 48 02 pm

Note:
To support color variations without having to depend on quiche components (coming soon), I created a tiny node script that iterates over all svg in icons folder, reads the contents of those files, substitutes fill attributes for each color variation we want to support and saves them into separate files. The script runs right before npm publish in circle in order to keep our source dir cleaner

### CODE REVIEW

File: svg-generator.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+
+if (process.env.NODE_ENV === 'deployment') {
+  const iconsDir = path.resolve(__dirname, 'src', 'icons');
+  const icons = fs.readdirSync(iconsDir);
+
+  const colorVariations = {
```
Comment (nhinsch) [inline svg-generator.js]: Should we import these colors from elsewhere in Juice?

**Comment by mattbriancon** [comment]
Why can't we use css to change the color of the stars? Also, when height, width, and viewBox are set like that svgs don't scale correctly.

File: svg-generator.js (hunk 1 above)
Comment (sashako) [inline svg-generator.js]: Juice is mainly scss files. It's not obvious to me how to pull in a sass map here. If you have any ideas, lmk

Comment (nhinsch) [inline svg-generator.js]: Makes sense :)

**Comment by sashako** [comment]
@mattbriancon Quiche components will do that. To have css control over svgs, svgs need to be inline. So that wouldn't really work for juice, which is a mixin library. The idea is, that most of the time icons will be used from Quiche. But I wanted to add juice-only support for colors as well, just in case. 
As for viewBox, it seems to scale fine. Sizing here is sone via background-size css property. All sizing info will most likely be completely stripped in quiche using inline-svg-loader

**Pull request merged**